lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
Added.

I quite like the generator btw, really helped me get everything up and
running quickly.

Already using this as the basis for my app, though I did have an issue
which perhaps I'll raise separately

On Thu, Nov 26, 2015 at 8:43 PM, Kevin Hughes <notifications@github.com>
wrote:

> In lib/sinatra/shopify-sinatra-app.rb
> <https://github.com/kevinhughes27/shopify-sinatra-app/pull/10#discussion_r46006063>
> :
>
> > @@ -189,7 +190,9 @@ def self.registered(app)
> >          provider :shopify,
> >                   app.settings.api_key,
> >                   app.settings.shared_secret,
> > -
> > +                 #Fixes for new Shopify callback issues
> > +                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
> > +                 callback_url: app.settings.hostname + "/auth/shopify/callback",
>
> Good point - can you add it here?
> https://github.com/kevinhughes27/shopify-sinatra-app/blob/master/bin/shopify-sinatra-app-generator#L42
>
> I am not really a fan of the generator in this repo and I might remove it
> soon but we might as well add it for now
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kevinhughes27/shopify-sinatra-app/pull/10/files#r46006063>
> .
>



-- 
Toby Green